config/pipeline_config/schema.json

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "object",
+  "required": [
+    "main_workflow_path",
+    "nf_config",
+    "nf_profile",
+    "environment",
+    "parameters"
+  ],
+  "properties": {
+    "main_workflow_path": {"type": "string"},
+    "nf_config": {"type": "string"},
+    "nf_profile": {"type": "string"},
+    "environment": {
+      "type": "object",
+      "additionalProperties": {"type": "string"}
+    },
+    "parameters": {
```

Comment:
If we call this `pipeline_parameters`, we could also have `nextflow_parameters` to easily pass extra nextflow parameters like `-with-tower` etc.

Reply:
Good point! Will fix, thanks for the feedback!